api/src/main/java/jakarta/mail/internet/MimeUtil.java

```diff
@@ -81,21 +79,13 @@ public static String cleanContentType(MimePart mp, String contentType) {
 
     /**
      * Convenience method to get our context class loader.
-     * Assert any privileges we might have and then call the
-     * Thread.getContextClassLoader method.
      */
     private static ClassLoader getContextClassLoader() {
-        return
-                AccessController.doPrivileged(new PrivilegedAction<ClassLoader>() {
-                    @Override
-                    public ClassLoader run() {
-                        ClassLoader cl = null;
-                        try {
-                            cl = Thread.currentThread().getContextClassLoader();
-                        } catch (SecurityException ex) {
-                        }
-                        return cl;
-                    }
-                });
+        ClassLoader cl = null;
+        try {
+            cl = Thread.currentThread().getContextClassLoader();
+        } catch (SecurityException ex) {
```

Comment:
This should be removed

Reply:
@jbescos I made the argument that this catch should remain for classes that can be subclasses. E.g. a thread that unconditionally throws SE. Since SE is not deprecated we should keep this fir compatibility.

Reply:
I see, thanks